ksqldb-serde/src/main/java/io/confluent/ksql/serde/protobuf/ProtobufNoSRConverter.java

```diff
@@ -44,6 +44,16 @@ public class ProtobufNoSRConverter implements Converter {
 
   private ProtobufData protobufData;
 
+  /**
+   * This constructor is defined only to adhere to the Converter plugin semantics of defining a
+   * no-arg constructor. The usage of this class is internal to ksqldb and is instantiated
+   * explicitly with Schema argument.
+   */
+  public ProtobufNoSRConverter() {
+    super();
```

Comment:
This line can be removed -- only need to call `super()` expliclity if we need to pass parameters.